// Вы должны вызвать функцию divide которая делит два числа, но она возвращает
// не только результат деления, но и информацию об ошибке.
// В случае какой-либо ошибки вы должны вывести "ошибка",
// если ошибки нет - результат функции. Функция divide(a int, b int) (int, error) получает
// на вход два числа которые нужно поделить и возвращает результат (int) и ошибку (error).
// Пакет main уже объявлен, а нужные пакеты уже импортированы!

// Не забудьте считать два числа которые необходимо поделить (передать в функцию)!

// Sample Input:   10 5
// Sample Output:  2

package main

import (
	"errors"
	"fmt"
)

func main() {
	var a int
	var b int
	_, err := fmt.Scan(&a, &b)
	if a == 0 || b == 0 {
		err := errors.New("ошибка")
		fmt.Println(err)
	} else if err != nil {
		fmt.Println("ошибка")
	} else {
		fmt.Printf("%d", a/b)
	}
}
